README.md

```diff
@@ -135,3 +135,49 @@ You may configure these in the repository secret settings on GitHub.
 To create such a certificate bundle, open Keychain Access. Unfold the entries for the development and distribution certificate. Select the certificates and their private keys using shift, then right-click and select "Export 4 items...".
 
 You can encode a file to base64 on the command line like this: `base64 -i ~/Desktop/Certificates.p12 | pbcopy`. This automatically puts the result on your clipboard.
+
+## Release process
+
+The process of releasing a new version of an app goes as follows.
+
+### Increment version number and commit
+
+First, a new version number is picked for the app. The version numbering convention is up to the project team to decide. [Semantic Versioning](https://semver.org/) may be used. For example, given a version number `MAJOR.MINOR.PATCH`, increment the:
+
+- MAJOR version when you make changes that impact the user's workflow of the app
+- MINOR version when you add functionality
+- PATCH version when you make bug fixes
+
+Set this version number in the Xcode project settings and commit it.
+
+### Create release branch
+
+Create a branch `release/<version>` in which `<version>` is the new version number.
+Push the branch to GitHub and create a pull request.
+GitHub Actions will automatically create a release build for you and upload it to App Store Connect. You can distribute this build to TestFlight if you want.
+
+Any last-minute changes before the release goes out may be committed to this branch.
```

Comment:
If we mention this, should we also mention merging this release branch back into the main branch after release?

Reply:
Sharp, I forgot to add that step